src/super_context.mli

```diff
@@ -53,6 +53,20 @@ val ocaml_flags
   -> Buildable.t
   -> Ocaml_flags.t
 
+val c_flags
```

Comment:
In fact, it would be better if these flags functions just took expanders. We're trying to minimize uses of Super_context as it's a large, and mostly meaningless module at this point.

Reply:
fixed.